### DISCUSSION TITLE
Auto Clicker Extension not working on next page until page is refreshed.

### DISCUSSION THREAD

**Discussion by dharmesh-hemaram** [Dhruv-Techapps/acf-docs#349] [Q&A] [answered]
We have two pages lets say `Product Page` and `Checkout Page`. on Selecting product from product page user is navigated to next page which is `checkout page` . But extension dont run on this page until I refresh Page or Press F5.

**Comment by dharmesh-hemaram** [comment, marked as answer]
Now a days many sites are created using Modern front end framework like React, Angular. Which loads page without reloading entire page this way it makes website faster by not loading all content from first and load only part of page / complete page behind the screen.

When a button is clicked and you see entirely new page with even URL is changed. But if all this process is done through framework website state is not refreshed to new page and its loading entire page into same container. 

> In above case. You need to write configuration for `checkout  page` also in `Product Page` configuration.

### Product Page Action
```
- URL : Product Page URL
- PAction - 1 - select size
- PAction - 2 - add to cart
```

### Checkout Page Action
```
- URL : Checkout Page URL
- CAction - 1 - select address
- CAction - 2 - checkout
```
> Merge them into single configuration


### Product and Checkout Page Action
```
- URL : Product Page URL
- PAction - 1 - select size
- PAction - 2 - add to cart
- CAction - 1 - select address ( Add init wait so action wait until checkout page is loaded)
- CAction - 2 - checkout
```

**Comment by mohamadho0sein** [comment]
I want Checkout Page Action to be done by your plugin
Product Page Action should be done manually by myself

- URL : Product Page URL
- CAction - 1 - select address ( Add init wait so action wait until checkout page is loaded)
- CAction - 2 - checkout

Is this formula correct?
